lib/utils/constants.dart

```diff
@@ -3,4 +3,4 @@ enum Mode {
   BUCKETING,
 }
 
-const version = "0.0.1";
+const version = "0.2.1";
```

Comment:
Well spotted ! I wonder if there is anyway we could make that automatic

Reply:
absolutely, we should

Reply:
the version is generated automatically.